core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/queue/MemoryQueue.scala

```diff
@@ -665,6 +679,9 @@ class MemoryQueue(private val etcdClient: EtcdClient,
 
       goto(Removed) using NoData()
     } else {
+      if (containers.size > 0) {
```

Comment:
If already has container for the remain activations, kill the monit actor to `don't create containers any more`